build.gradle

```diff
@@ -535,6 +541,24 @@ configure(subprojects - getJsProjects()) { subproject ->
     }
 }
 
+def checkDexer = task('checkDexer', description: 'compiles android class files into dex bytecode', dependsOn: compileKotlin) {
+    getAndroidProjects().each { p->
```

Comment:
That's a nice idea but in the end I would like to have not only a task on the root of the project but individual tasks for those projects, so that I can do: `gradlew :atrium-core-api-android:checkDexer`

Hence the advice to do:
```
configure(getAndroidProjects()) { p ->
...
}
```